<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>FAÇA SEU LOGIN:</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>9</number>
    </property>
    <property name="verticalSpacing">
     <number>14</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <item row="0" column="1">
     <widget class="QLineEdit" name="usernameText"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Usuário</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Senha</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="passwordText">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>usernameText</zorder>
   <zorder>passwordText</zorder>
  </widget>
  <widget class="QPushButton" name="botaoLogin">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>160</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QWidget" name="fundo" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>241</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>441</width>
     <height>241</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#fundo
{background-color: qlineargradient(spread:pad, x1:0.512, y1:0, x2:0.518, y2:1, stop:0 rgba(254, 255, 172, 255), stop:1 rgba(255, 170, 127, 255));}
</string>
   </property>
  </widget>
  <zorder>fundo</zorder>
  <zorder>label</zorder>
  <zorder>formLayoutWidget</zorder>
  <zorder>botaoLogin</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
